eclipse/mars/gcp-eclipse-mars.tpd

```diff
@@ -18,6 +21,7 @@ location "http://download.eclipse.org/cbi/updates/license" {
 location "http://download.eclipse.org/releases/mars/" {
 	org.eclipse.sdk.feature.group
 	org.eclipse.jdt.feature.group
+  org.eclipse.platform.feature.group
```

Comment:
Huh, why did you need to bring this in?  It should be included with org.eclipse.sdk